src/main/java/com/epam/ta/reportportal/database/entity/settings/SamlProviderDetails.java

```diff
@@ -0,0 +1,109 @@
+package com.epam.ta.reportportal.database.entity.settings;
```

Comment:
Could you please add the copyright header as in other classes?

Reply:
Added licence to the file header